pkg/api/acknowledge.go

```diff
@@ -68,26 +68,28 @@ func (a *API) acknowledge(message slack.InteractionCallback) error {
 	}
 
 	f := &clients.Filter{}
-	if f.ClusterFilterFromText(message.OriginalMessage.Attachments[0].Text) != nil ||
-	   f.AlertnameFilterFromText(message.OriginalMessage.Attachments[0].Text) !=nil {
-		return errors.New("slack message parsing for alertname and cluster failed")
-	}
-
-	incident, err := a.pdClient.GetIncident(f)
-	if err != nil {
-		return err
-	}
+	for  i := 0; i < len(message.OriginalMessage.Attachments); i++ {
```

Comment:
please use go's range `for _, attachments := range message.OriginalMessage.Attachments { .. }`

Reply:
done